site/docs/skylark/language.md

```diff
@@ -105,10 +105,9 @@ illegal, and 2) `*args` and `**kwargs` arguments are not allowed.
 * Global variables are immutable.
 
 * `for` statements are not allowed at the top-level. Use them within functions
-  instead. In BUILD files, you may use list comprehensions.
+  instead. In BUILD files, you may use list comprehensions. That said, we prefer that you use macros, because we think they are easier to read.
```

Comment:
I'd remove this line (I know it was suggested by @brandjon).
We also want to limit the use of macros.

Reply:
I removed it. Thank you!